release-team/role-handbooks/ci-signal/README.md

```diff
@@ -1,169 +1,181 @@
+-   Availability to attend the majority of Release Team (weekly) and Burndown meetings (daily during Code Freeze), subject to time zone appropriateness.
+-   Ability to follow-up on test fixes during Code Freeze at arbitrary times to ensure rapid turnaround.
+-   The time commitment becomes greater through the release cycle, peaking during Code Freeze. In the last week of Code Freeze, Shadows should expect to spend at least 10 hours and leads at least 20 hours.
 
 ### Additional Requirements for Shadows
 
 The following requirements are all things that Leads should already have, and Shadows should acquire, with coaching, within the first weeks of the release cycle:
 
-- Have signed the contributor CLA for Kubernetes.
-- Become a Kubernetes org member.  In many cases, this will be done with the sponsorship of the CI Signal Lead or Release Lead in the first week of the cycle.
-  - The process to become one of these is in [our community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-for-outside-collaborators)
-- General familiarity with our [test boards](https://testgrid.k8s.io/) and examining test results from automated tests.
-- Willingness and ability to follow-up with contributors about test failures, on Slack, email, Discuss, and SIG meetings, as appropriate.
-- Ability to file well-crafted Kubernetes issues, including labelling.
-- General knowledge of our SIG system and SIGs' areas of responsibility.
+-   Have signed the contributor CLA for Kubernetes.
+-   Become a Kubernetes org member. In many cases, this will be done with the sponsorship of the CI Signal Lead or Release Lead in the first week of the cycle.
+    -   The process to become one of these is in [our community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-for-outside-collaborators)
+-   General familiarity with our [test boards](https://testgrid.k8s.io/) and examining test results from automated tests.
+-   Willingness and ability to follow-up with contributors about test failures, on Slack, email, Discuss, and SIG meetings, as appropriate.
+-   Ability to file well-crafted Kubernetes issues, including labelling.
+-   General knowledge of our SIG system and SIGs' areas of responsibility.
 
 Additionally, the following qualifications make a candidate more suitable for the CI Signal team, but are not requirements:
 
-- Prior involvement with SIG Testing and the Test Infrastructure team.
-- Experience with automated testing, CI/CD, quality engineering, and/or quality assurance.
+-   Prior involvement with SIG Testing and the Test Infrastructure team.
+-   Experience with automated testing, CI/CD, quality engineering, and/or quality assurance.
+-   Experience with Golang, as the CI Signal team reviews the [CI Signal reporting tool](#reporting-status) written in Golang.
```

Comment:
Maybe more like
"A basic understanding of Golang to review and improve the CI Signal reporting tool"